### ISSUE TITLE
Label smoothing has no effect

### ISSUE THREAD

**Issue by ligrire** [omihub777/ViT-CIFAR#2]
In utils.py line 11:
```
        if args.label_smoothing:
            criterion = LabelSmoothingCrossEntropyLoss(args.num_classes, smoothing=args.smoothing)
        criterion = nn.CrossEntropyLoss()
```
The criterion will be `nn.CrossEntropyLoss()` anyway regardless of the argument of `label_smoothing` because there is no else statement.

**Comment by omihub777** [comment]
Fixed
